src/App.jsx

```diff
@@ -14,45 +16,102 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+export const App = () => {
+  const [isReversed, setIsreversed] = useState(false);
+  const [sortType, setSortType] = useState('');
+  const SORT_BY_ALPHABET = 'alphabet';
+  const SORT_BY_LENGTH = 'length';
+
+  function getSortedGoods(goods, sortField) {
```

Comment:
Consider moving the function outside from the App component for better performance